sparkplug-core/src/java/sparkplug/function/ComparatorFn.java

```diff
@@ -11,8 +12,8 @@
  */
 public class ComparatorFn extends SerializableFn implements Comparator<Object> {
 
-    public ComparatorFn(IFn f) {
-        super(f);
+    public ComparatorFn(IFn f, Collection<String> namespaces) {
```

Comment:
Very minor but I think `Set<String>` instead of `Collection<String>` is a slightly better interface because I'm pretty sure we want them to always be unique namespaces.